src/slack/blocks.js

```diff
@@ -0,0 +1,64 @@
+// block-building helpers used in several bots now - invoicer, payroll-bot, coffee-bot, askmebot, ...
```

Comment:
Then it might be a good idea to create it as a separate (private) package in ghcr.io 😄

Anyway, I'm just leaving this note here, probably if someone can handle it later can take a stab at it.

Reply:
definitely a good idea. though the end game around bots is to move them together into one repo, so it will be even easier